plugins/obs-ffmpeg/data/locale/en-US.ini

```diff
@@ -49,6 +49,9 @@ Play="Play"
 Pause="Pause"
 Stop="Stop"
 
+FFmpeg.Directives="FFmpeg Directives"
```

Comment:
As far as I can tell, FFmpeg refers to these as "options".  See:
* https://ffmpeg.org/ffmpeg.html#Options.
* https://ffmpeg.org/ffmpeg-protocols.html#Protocol-Options
* https://ffmpeg.org/ffmpeg-protocols.html#rtsp

FFmpeg uses the term "directives" to refer to something else entirely (the concat demuxer).  I'd prefer keeping the terminology consistent so that it's searchable.

Reply:
done